scripts/codemod/renameCmp.js

```diff
@@ -0,0 +1,89 @@
+// @flow strict
+
+/**
+ * CODEMOD to RENAME GESTALT COMPONENTS
+ * Ex. <Flyout /> to <Popover />
+ *
+ * TO RUN THIS CODEMOD
+ * yarn run:codemod renameCmp  <folder/file path> --previousCmpName=<value> --nextCmpName=<value>
```

Comment:
@AlbertCarreras could we make this `yarn codemod renameComponent`?

Reply:
DONE